starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -2148,14 +2151,10 @@ sub print_report {
     $o .= "\n";
 
     if (exists $self->{figure_of_merit}) {
-	my @probs = @{ $self->{figure_of_merit}->{cum_prob}};
 	my $bad_FOM = $self->{figure_of_merit}->{cum_prob_bad};
 	$o .= "$red_font_start" if $bad_FOM;
-	$o .= "Probability of acquiring 2,3, and 4 or fewer stars (10^x):\t";
-        # override formatting to match pre-JSON strings
-        foreach (2..4) {
-            $o .= substr(sprintf("%.4f", "$probs[$_]"), 0, 6) . "\t";
-        }
+	$o .= "Probability of acquiring 2 or fewer stars (10^x):\t";
```

Comment:
I think we should be printing out `P2`, i.e. `-log10(..)`.  So this would change to `(10^-x)`.